grunt/aliases.yml

```diff
@@ -8,12 +8,14 @@ build-dev:
 
 build:
   - makepot
```

Comment:
Could you explain the why behind these changes, just so we have a record of it on the PR.